src/status_im/ui/screens/browser/views.cljs

```diff
@@ -0,0 +1,99 @@
+(ns status-im.ui.screens.browser.views
+  (:require-macros [status-im.utils.views :as views])
+  (:require [re-frame.core :as re-frame]
+            [status-im.ui.components.react :as react]
+            [status-im.ui.screens.browser.styles :as styles]
+            [status-im.ui.components.status-bar.view :as status-bar.view]
+            [status-im.ui.components.toolbar.view :as toolbar.view]
+            [status-im.chat.views.toolbar-content :as toolbar-content]
+            [status-im.ui.components.webview-bridge :as components.webview-bridge]
+            [status-im.utils.js-resources :as js-res]
+            [status-im.ui.components.react :as components]
+            [reagent.core :as reagent]
+            [status-im.ui.components.chat-icon.screen :as chat-icon.screen]
+            [status-im.ui.components.icons.vector-icons :as vector-icons]
+            [status-im.i18n :as i18n]))
+
+(views/defview toolbar-content-dapp [contact]
+  (views/letsubs [contact' [:contact-by-identity contact]]
```

Comment:
First contact could be `id`, then no need for `contact'`

Reply:
yeah